src/documents/TextToSpeechAPI.md

```diff
@@ -471,9 +478,16 @@ fluid.textToSpeech({
             <td>
 <pre>
 <code>
+
+// Assuming an existing textToSpeech component
+var availableVoices = textToSpeech.getVoices();
+var voiceToUse = availableVoices[0];
```

Comment:
We may want to include a JIRA number related to modifying the options to take in a voice identifier.